_pages/resources/training-material.md

```diff
@@ -7,4 +7,31 @@ sidebar:
   nav: "uhpc_menu"
 ---
 
-We will upload our training material very soon.
+## Course Material
+
+All the material is hosted [here](https://github.com/UNIVERSE-HPC/course-material) written in github flavored markdown with a few custom directives to indicate callouts, problem and solutions blocks. Each section also has its own metadata header section in YAML, which contains information on the name, the dependencies of each section, and the attribution information for that section of material. The format is described in more detail in the [CONTRIBUTING.md](https://github.com/UNIVERSE-HPC/course-material/blob/main/CONTRIBUTING.md)
+
+## Course website
+
+To render the material in a form suitable for running courses, we have written a website called [Gutenberg](https://github.com/OxfordRSE/gutenberg). You can see an example deployment of this website by Oxford Research Software Engineering (OxRSE) [here](https://train.oxrse.uk/). The website renders the material as static webpages, but also has a database and backend for hosting course "events". Events are comprised as a linear series of modules along with timing and location information and pointers to material. Students can enrol on courses, and tick off problems as they complete them, enabling instructors to track progress. Students can also comment directly on the material in order to obtain asynchronous, remote feedback from other students and instructors if required. 
+
+## Courses and Modules
+
+| Course | Module | Description |
+| --- | --- | --- |
+| [Software Architecture and Design](https://train.oxrse.uk/material/software_architecture_and_design) | [Procedural Programming](https://train.oxrse.uk/material/software_architecture_and_design/procedural) | Procedural Programming is based around the idea that code should be structured into a sequence of **procedures** that operate on data. This course will introduce you to the basics of procedural programming in either Python or C++.  |
+| | [Object-Orientated Programming](https://train.oxrse.uk/material/software_architecture_and_design/object_orientated) | The Object Oriented Paradigm builds upon the Procedural Paradigm, but builds code around data. This course will introduce you to the basics of Object Oriented Programming in either Python or C++.  |
```

Comment:
Maybe say something about OO not just building code around data but also problem structure? If my suggested change confuses things, feel free to ignore.

```suggestion
| | [Object-Orientated Programming](https://train.oxrse.uk/material/software_architecture_and_design/object_orientated) | The Object Oriented Paradigm builds upon the Procedural Paradigm, but builds code around data and the structure of the problem being solved. This course will introduce you to the basics of Object Oriented Programming in either Python or C++.  |
```